saleor/graphql/core/scalars.py

```diff
@@ -209,3 +209,28 @@ class Hour(graphene.Int):
 
 class Day(graphene.Int):
     """The `Day` scalar type represents number of days by integer value."""
+
+
+class PositiveInt(graphene.Int):
+    """Nonnegative Integer scalar implementation.
```

Comment:
is it PositiveInt or NonNegativeInt?  Also, `parse_` methods  exclude 0, while description says 0 is ok